modules/react/combobox/lib/hooks/useComboboxKeyboardTypeAhead.ts

```diff
@@ -57,13 +57,13 @@ export const useComboboxKeyboardTypeAhead = createElemPropsHook(useComboboxModel
     return -1;
   };
 
-  const currentItem = model.navigation.getItem(model.state.cursorId, model);
-  const cursorFocusedIndex = currentItem.index;
+  const currentItem =
+    model.state.items.length > 0 ? model.navigation.getItem(model.state.cursorId, model).index : 0;
 
   const handleKeyboardTypeAhead = (key: string, numOptions: number) => {
     // If the starting point is beyond the list of options, reset it
     // to the beginning of the list
-    const startNumber = keySofar.current.length === 0 ? cursorFocusedIndex + 1 : cursorFocusedIndex;
+    const startNumber = keySofar.current.length === 0 ? currentItem + 1 : currentItem;
```

Comment:
This initially confused me. `currentItem` is being assigned to an index. Then, looking above, `currentItem` was an item, but now changed to an index. It would be more clear if the constant was renamed to `currentItemIndex`.